.github/workflows/golvulncheck.yml

```diff
@@ -0,0 +1,33 @@
+name: Run govulncheck
+on:
+  push:
+  pull_request:
+
+env:
+  GO_VERSION: "1.22"
+  USE_BROTLI: 1
+  USE_LIBSODIUM: 1
+  USE_LZO: 1
+
+jobs:
+  scan:
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Install deps
+        run: sudo apt-get install liblzo2-dev brotli libsodium-dev
+
+      - name: Set up Go 1.x
+        uses: actions/setup-go@v5
+        with:
+          go-version: ${{ env.GO_VERSION }}
+        id: go
+
+      - name: Get dependencies
+        run: |
+          go get -v -t -d ./...
+      - id: govulncheck
+        uses: golang/govulncheck-action@v1
```

Comment:
It looks like it produces a lot of irrelevant to PR errors.
I will suggest adding `continue-on-error: true` for this step?

Reply:
it will not change things. if those errors are ignored now in some way, they will be ignored in another way.

the idea is to fix those findings first. after that it is ok to merge